templates/post_detail.html

```diff
@@ -6,8 +6,17 @@
   
   <h1>{{post.title}} </h1>
   
-  <h1>{{post.cover}}</h1>
-  <!-- <h1>{{post.owner}}</h1> -->
+  <img src="{{ post.cover.url}}" alt="{{ post.title }}">
+  
+  <h3>Number of Likes: {{post.like_set.all.count}}</h3>
```

Comment:
[suggestion]
You *might* be able to do `post.like_set.count` here, and bypass the `all` manager, but I'm not sure of that. Otherwise, this is an excellent use of django's ORM!